docs/scout.txt

```diff
@@ -0,0 +1,198 @@
+.. _laravel-scout:
+
+============================
+Integrated Search with Scout
+============================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: php framework, odm, code example, text search, atlas
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to use the Laravel Scout feature in
+your {+odm-long+} application. Scout allows you to implement full-text
+search on your Eloquent models. To learn more, see `Laravel Scout
+<https://laravel.com/docs/{+laravel-docs-version+}/scout>`__ in the
+Laravel documentation.
+
+The Scout Integration for {+odm-long+} provides the following
+functionality:
+
+- Provides an abstraction to create search indexes on documents in
+  MongoDB collections and other search engines. In MongoDB, this feature
+  allows you to create :atlas:`Atlas Search indexes
+  </atlas-search/manage-indexes/>`.
+
+- Allows you to automatically replicate data from MongoDB into a
+  search engine such as `Meilisearch <https://www.meilisearch.com/>`__
+  or `Algolia <https://www.algolia.com/>`__. You can use a MongoDB Eloquent
+  model as the source to import and index.
+
+.. note:: Deployment Compatibility
+
+   You can use Laravel Scout only when you connect to MongoDB Atlas
+   clusters. This feature is not available for self-managed or
+   serverless deployments.
+
+Install Scout Package
+---------------------
+
+Before you can use Scout in your application, you must run the following
+command to install Laravel Scout:
+
+.. code-block:: bash
+   
+   composer require laravel/scout
+
+Add the Searchable Trait to Your Model
+--------------------------------------
+
+Add the ``Laravel\Scout\Searchable`` trait to an Eloquent model to make
+it searchable. This example uses the ``Movie`` model, which represents
+documents in the ``sample_mflix.movies`` collection.
```

Comment:
S: you can make this sentence more of an introduction/slightly more descriptive of the code:
```suggestion
it searchable. The following example adds this trait to the ``Movie`` model, which represents
documents in the ``sample_mflix.movies`` collection:
```